### PR TITLE
IN_1891 Create views.view.ord_staff.yml

### PR DESCRIPTION
**Pull request by jdearie**

### CODE REVIEW

File: config/sync/views.view.ord_staff.yml
Comment (jmarruff) [inline config/sync/views.view.ord_staff.yml:1]: Did we want to change the id, label, and description by removing the ORD designation?

Comment (jmarruff) [reply config/sync/views.view.ord_staff.yml:1]: Lines 17, 18, and 20.

**Review by jmarruff — CHANGES_REQUESTED** [review]
updating lines 17, 18, and 20?

File: config/sync/views.view.ord_staff.yml
Comment (jdearie) [reply config/sync/views.view.ord_staff.yml:1]: updated view display name and description

**Review by jmarruff — APPROVED** [review]

**Pull request merged**